pkg/services/sqlstore/migrations/ualert/rule_group_index_migration.go

```diff
@@ -0,0 +1,209 @@
+package ualert
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"xorm.io/xorm"
+
+	ngmodels "github.com/grafana/grafana/pkg/services/ngalert/models"
+	"github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+)
+
+// UpdateRuleGroupIndexMigration updates a new field rule_group_index for alert rules that belong to a group with more than 1 alert.
+func UpdateRuleGroupIndexMigration(mg *migrator.Migrator) {
+	mg.AddMigration("update group index for alert rules", &updateRulesOrderInGroup{})
```

Comment:
Moved as-is from ualert along with the vendored structs it needed from elsewhere in the migration model.

The only change is here is removing the [condition to only add if UA is enabled.](https://github.com/grafana/grafana/blob/58f6648505d1fbc05d6fc9ee0b37720a9f3c3f5d/pkg/services/sqlstore/migrations/ualert/ualert.go#L807-L809)